src/components/GraphVisualization/GraphComponent.tsx

```diff
@@ -18,18 +18,20 @@ export interface GraphData {
 const graphConfig: Partial<GraphConfiguration<GraphNode, GraphLink>> = {
   nodeHighlightBehavior: true,
   node: {
-    color: "lightgray",
-    highlightColor: "orange",
+    color: "#eeeeee",
+    highlightColor: "#FAB900",
     size: 200,
+    fontColor: "#36454c",
   },
   link: {
-    highlightColor: "orange",
+    highlightColor: "#FAB900",
   },
+  height: 550,
```

Comment:
to żeby graf był na całym polu - wiem, że brzydko zafiksowane :/

Reply:
Super!
Tylko czy mogłabyś też zmienić szerokość?
![image](https://user-images.githubusercontent.com/21079319/121814003-0ad37b00-cc6f-11eb-9c7a-b1e79d4c07bb.png)

Reply:
zmieniłam, teraz jest na window.innerWidth / 2
nie umiem na szybko lepiej, ale teraz będzie uciekać poza pole w prawo :(